lib/rules/no-duplicate-selectors/index.js

```diff
@@ -63,19 +63,13 @@ const rule = (primary, secondaryOptions) => {
 			);
 			const resolvedSelectorList = [
 				...new Set(
-					ruleNode.selectors.flatMap((selector) => {
-						return isStandardSyntaxSelector(selector)
-							? resolvedNestedSelector(selector, ruleNode)
-							: [];
-					}),
+					ruleNode.selectors.flatMap((selector) => resolvedNestedSelector(selector, ruleNode)),
 				),
 			];
 
-			if (resolvedSelectorList.length === 0) {
-				return;
-			}
-
-			const normalizedSelectorList = resolvedSelectorList.map(normalize);
+			const normalizedSelectorList = resolvedSelectorList.map((selector) => {
+				return isStandardSyntaxSelector(selector) ? normalize(selector) : selector;
```

Comment:
We should not filter them out, because that would change the combined selector semantic.